src/components/vr_module/include/vr_module/service_module.h

```diff
@@ -95,6 +95,12 @@ class ServiceModule {
   virtual void DeactivateService() = 0;
 
   /**
+   * Checks if one of the services is activated
+   * @return true if one of the services is activated
+   */
+  virtual bool HasActivatedService() const = 0;
```

Comment:
Sorry, I didn't get you question.

I meant that if it's check if service has activated it will be better to use name HasServiceActivated. 
But if it checks does VRModule has any activated services your name fully correct :)